src/App.vue

```diff
@@ -165,6 +165,11 @@ export default {
     window.removeEventListener("beforeunload", this.onClose);
   },
   mounted() {
+    // remove hash from the url if it is present
+    if (location.hash) {
+      location.replace(location.hash.replace("#", ""));
+    }
```

Comment:
Is it possible to write a test for this?

Reply:
<img width="515" alt="Screenshot 2021-11-05 at 12 24 54 PM" src="https://user-images.githubusercontent.com/13185699/140470461-fb65318e-5909-4414-b899-3835bffb8b4d.png">
I don't think so - as `location` is a property of the `window` object. For all our tests, we've had to define mock functions for `window`'s properties. So, we'll have to write the implementation of `location.hash`, `location.replace` and we can only test our implementation.